tests/integration/test_charm.py

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env python3
+# Copyright 2023 Canonical Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Test slurmdbd charm against other SLURM charms in the latest/edge channel."""
+
+import asyncio
+import logging
+import pathlib
+from typing import Any, Coroutine
+
+import pytest
+import tenacity
+from helpers import get_slurmctld_res, get_slurmdbd_res
+from pytest_operator.plugin import OpsTest
+
+logger = logging.getLogger(__name__)
+
+SERIES = ["focal"]
+SLURMDBD = "slurmdbd"
+SLURMCTLD = "slurmctld"
+DATABASE = "percona-cluster"
```

Comment:
Hi @NucciTheBoss! When do you plan to replace` percona-cluster`?

Reply:
The migration work will start later next week after I get back from my trip to Belgium. I know what needs to be changed within the slurmdbd operator to work with the mysql router and standalone mysql charm, it's mostly just writing the complementary unit and integration tests to make sure that my changes actually work :sweat_smile:

Last thing that David and I want to get out of the way before the migration is a GitHub charm publishing workflow for the SLURM charms so that the latest changes will be published to the edge channel on Charmhub.

Reply:
Excellent! Thank you!